code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -494,6 +494,7 @@ Difficulty: Hard
 	var/currently_seeking = FALSE
 	var/friendly_fire_check = FALSE //if blasts produced apply friendly fire
 	var/monster_damage_boost = TRUE
+	var/blast_type = /obj/effect/temp_visual/hierophant/blast
```

Comment:
I should actually probably just remove this entirely since I'm not using them anymore, come to think of it